### ISSUE TITLE
[Enhancement] Clarity of 'stored keywords'

### ISSUE THREAD

**Issue by ator-dev** [searchmarkers/mark-my-search#84]
![image](https://user-images.githubusercontent.com/66553618/211526240-5a5c12f1-352b-4cee-b404-63e3f760022e.png)

The concept of 'stored keywords' has existed for a while now (and replaced a stranger interface to the same system), but I've never properly gathered feedback on whether its reason for existing is clear. I'm concerned that it may be confusing to users and deter them from using the popup.

If anyone could tell me their understanding of what 'stored keywords' are (and so what 'restore keywords in tabs' is for), that would be greatly appreciated!

**Comment by ator-dev** [comment]
@buckleyGI I remembered about this because of your usability issue with the pin button, and I thought you might have an idea about this too - any input would be really helpful if you have the time.

P.S. This is the only time I'll mention you out of the blue like this. Thanks for your input on pinned keywords (:

**Comment by buckleyGI** [comment]
At the moment I have no clue what 'stored keywords' are. I'll keep it in mind when using the extension the coming days.

**Comment by buckleyGI** [comment]
And no problem that you mention me :)

**Comment by ator-dev** [comment]
Planning to make renames for clarity:
- "Restore keywords in tabs" to "Restore on reactivation" or similar
- "Keywords stored" to "Will restore last keywords"

If there are no keywords when MMS is deactivated in a tab, the empty array will be deleted so the "Keywords stored"/"Will restore last keywords" option is deactivated as it would have no effect anyway (would not as such 'restore keywords'). This should make it more intuitive.

**Comment by ator-dev** [comment]
I realised that more of the popup is unclear than I had previously thought. In light of this I have redesigned its entire landing page for accessibility to less technically-inclined users.

<details><summary>Previous popup UI, resulting from months of adding to a crowded interface</summary>

![image](https://user-images.githubusercontent.com/66553618/224542472-35e28f44-32b3-4d4f-b2ab-d0e83df280c9.png)

</details>

Refreshed UI

![image](https://user-images.githubusercontent.com/66553618/224542450-9064814f-4dfe-4bc9-a391-0d5d64cc6f05.png)

Most significant changes:
- "Follow links" is permanently on; there is no setting to turn it off. I cannot think of any use case where the behaviour of carrying highlights into new tabs would be unwanted.
- "Keywords stored" is now a button "Delete keywords" with a (more) obvious action. Clicking it without "Active" will mean keywords are not restored on next activating, regardless of "Restore keywords on reactivation". Clicking it with "Active" will remove the highlighting AND mean they are not restored.
- Other areas are clarified.
